src/flow.h

```diff
@@ -329,6 +329,9 @@ typedef struct Flow_
     /** flow queue id, used with autofp */
     SC_ATOMIC_DECLARE(int16_t, autofp_tmqh_flow_qid);
 
+    /** flow hash - the flow hash before hash table size mod. */
+    uint32_t flow_hash;
```

Comment:
it may actually be a good idea to move the startts field up there as well, I think that would make the header fit perfectly into it's own cache line.